docs/resources/samples/report-day-to-day-changes.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Record day-to-day changes in a workbook and report them with a flow
+description: Learn how to use Office Scripts and Power Automate to track value changes in a workbook
+ms.date: 08/25/2022
+ms.localizationpriority: medium
+---
+
+# Record day-to-day changes in a workbook and report them with a flow
+
+Power Automate and Office Scripts combine to handle repetitive tasks for you. In this sample, you're tasked with recording a single numerical reading in a workbook every day and reporting the change since yesterday. You'll build a flow to get that reading, log it in the workbook, and report the change through an email.
+
+## Sample Excel file
+
+Download <a href="daily-readings.xlsx">daily-readings.xlsx</a> for a ready-to-use workbook. Add the following script to try the sample yourself!
```

Comment:
```suggestion
Download [daily-readings.xlsx](daily-readings.xlsx) for a ready-to-use workbook. Add the following script to try the sample yourself!
```
Use markdown instead of HTML for links. Please verify this works!

Reply:
This didn't used to work for file downloads, but seems to now. Excellent!